contracts/erc20-gateway/ERC20Cogateway.sol

```diff
@@ -339,6 +355,8 @@ contract ERC20Cogateway is
         UtilityTokenInterface(utilityToken).mint(msg.sender, feeAmount);
 
         UtilityTokenInterface(utilityToken).mint(_beneficiary, mintAmount);
+
+        emit DepositIntentConfirmed(messageHash_);
```

Comment:
Please, update documentation accordingly (it currently says `Emits `WithdrawIntentConfirmed` event with the `messageHash_` parameter.`).

Reply:
It is updated now.